modelling/src/main/java/org/axonframework/modelling/entity/child/AbstractEntityChildModel.java

```diff
@@ -142,35 +141,47 @@ protected Builder(@Nonnull Class<P> parentClass,
-         * {@link ChildAmbiguityException} is thrown.
+         * Sets the {@link CommandTargetResolver} to use for resolving the child entity to handle the command. This
+         * should return one child entity, or {@code null} if no child entity should handle the command.
+         * <p>
+         * Defaults to matching a singular child entity, or throwing a {@link ChildAmbiguityException} if more than one
+         * child candidate exists. As such, provide another implementation if there are multiple candidates, such as
+         * when the member is a {@link List} of child entities.
          *
-         * @param commandTargetMatcher The {@link ChildEntityMatcher} to use for matching the child entities to the
-         *                             command.
+         * @param commandTargetResolver The {@link CommandTargetResolver} to use for resolving the child entity
+         *                              to handle the command.
          * @return This builder instance.
          */
         @SuppressWarnings("unchecked")
-        public R commandTargetMatcher(
-                @Nonnull ChildEntityMatcher<C, CommandMessage<?>> commandTargetMatcher) {
-            this.commandTargetMatcher = requireNonNull(commandTargetMatcher,
-                                                       "The commandTargetMatcher may not be null.");
+        public R commandTargetResolver(@Nonnull CommandTargetResolver<C> commandTargetResolver) {
+            this.commandTargetResolver = requireNonNull(commandTargetResolver,
+                                                        "The commandTargetResolver may not be null.");
             return (R) this;
         }
 
+        protected void validate() {
+            if (commandTargetResolver == null) {
+                throw new AxonConfigurationException("The commandTargetResolver must be set before building the model.");
+            }
+            if (eventTargetMatcher == null) {
+                throw new AxonConfigurationException("The eventTargetMatcher must be set before building the model.");
+            }
+        }
+
         /**
-         * Sets the {@link ChildEntityMatcher} to use for matching the child entities to the event. This is used to
-         * filter the child entities based on the event. The amount of entities that match an incoming event is not
-         * restricted. Entities that match are evolved, while entities that do not match are ignored and remain in the
-         * same state.
+         * Sets the {@link EventTargetMatcher} to determine whether a child entity should handle the given
+         * {@link EventMessage}. This should return {@code true} if the child entity should handle the event, or
+         * {@code false} if it should not.
+         * <p>
+         * Defaults to matching any child entity, meaning all child entities represented by this model will be evolved
```

Comment:
I don't think this has a default. Only the `SingleEntityChildModel` has a default.